ostia-operator/examples/simpleRateLimit.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: "ostia.3scale.net/v1alpha1"
+kind: "API"
+metadata:
+  name: "hello"
+spec:
+    expose: true
+    hostname: hello.10.32.241.228.nip.io
+    endpoints:
+        - name: hello
+          host: https://echo-api.3scale.net
+          path: /hello
+    rate_limits:
+        - service: hello
+          type: fixed
+          max_request: 1
+          time_interval: 10
+          time_unit: minutes
```

Comment:
this looks like 1 request for each 10 minutes, not 10_000 requests per minute, right?

Reply:
Yes correct, I was just testing the minutes functionality I had added and forgot to revert it. Will fix now.